src/pages/components/leadspace.mdx

```diff
@@ -29,7 +28,23 @@ import ComponentFeedback from "components/ComponentFeedback";
 
 This is the most commonly used Lead space style. The left-aligned type draws the users' attention to the Lead space messaging immediately. Used in marketing and product pages, among others.
 
-![Lead space left-aligned](../../images/pattern/leadspace/Leadspace_Demo_Centered.jpg)
+There are four available height sizes:
+
+- Super (640px)
+- Tall (560px)
+- Medium (480px)
+- Short (320px)
+
+For detailed guidelines around usage, also reference the [Lead space pattern page](/patterns/lead-space).
```

Comment:
"For specific usage of each height, please [see Lead space pattern](/patterns/lead-space) page. "

Generally we should make sure <a> link wrap the the Call to action - always try to start with a verb / action. Good for accessibility. Imagine screen reader voice.